cookbooks/bcpc/attributes/graphite.rb

```diff
@@ -2,8 +2,10 @@
 default['bcpc']['graphite']['relay_port'] = 2013
 default['bcpc']['graphite']['web_port'] = 8888
 default['bcpc']['graphite']['log']['retention'] = 15
+default['bcpc']['graphite']['data']['retention'] = 15
 default['bcpc']['graphite']['timezone'] = "'America/New_York'"
 default['bcpc']['graphite']['local_data_dir'] = "/opt/graphite/storage/whisper"
+default['bcpc']['graphite']['local_log_dir'] = "/opt/graphite/storage/log"
```

Comment:
If you're adding this, could you [update](https://github.com/kiiranh/chef-bach/blob/2e49aec106fea4799bfa0a66f0a5bbd20c7f0005/cookbooks/bcpc/templates/default/graphite.local_settings.py.erb#L73) Graphite to know that is where its logs should go?